client/post-editor/editor-more-options/copy-post.jsx

```diff
@@ -66,6 +66,11 @@ class EditorMoreOptionsCopyPost extends Component {
 
 	render() {
 		const { translate, siteId } = this.props;
+
+		if ( ! siteId ) {
```

Comment:
The idea is that, since this option is quite hidden (last option of the last sidebar drawer) and hardly the user will rush for it in the short time when the site info is missing, there would be no need to render it immediately.